third-party/CMakeLists.txt

```diff
@@ -75,6 +75,13 @@ if(UNIX)
   else()
     message(STATUS "Found GNU Make at ${MAKE_PRG}")
   endif()
+  find_program(PATCH_PRG patch)
```

Comment:
git is required and used by `BuildLibvterm.cmake` etc. So this should not be needed: we always required git to build on Windows.